CONTRIBUTING.adoc

```diff
@@ -99,100 +154,178 @@ Once your author file is defined, you can return to your blog post file
+  Documentation.
+
+
+==== Jenkins User Documentation
+
+The Jenkins User Documentation consists of the following parts:
+
+* *Tutorials* - these are step-by-step guides that teach users relatively new to
+  Continuous Integration (CI) / Continuous Delivery (CD) concepts about how to
+  implement their project (of a particular tech stack) in Jenkins. A tutorial's
+  content is based on the "tutorial" description in Daniele Procida's blog post
+  "link:https://www.divio.com/en/blog/documentation/[What nobody tells you about
+  documentation]". Read more about
+  <<adding-a-tutorial-page,Adding a Tutorial page>>.
+* *How-to guides* - these are short guides consisting of procedures to get the
+  reader started with specific/common use-case scenarios. They could also be
+  guides that assist with overcoming commonly encountered issues - thereby
+  behaving as a form of knowledgebase article. A how-to guide's content goes
+  beyond the more general scope of a topic in the User Handbook, but these
+  guides do not hand-hold or teach the reader using very specific scenarios
+  (e.g. forking a given repo), as the *Tutorials* do. A how-to guide's content
+  is based on the "how-to guide" description in Daniele Procida's blog post
+  "link:https://www.divio.com/en/blog/documentation/[What nobody tells you about
+  documentation]". While there are currently no "how-to guides" yet, this
+  section will be added when good candidate guides arise.
+* *User Handbook* - rich and in-depth documentation, separated into chapters,
+  each of which covers a given topic/feature of Jenkins. This is conceptually
+  and structurally similar to the
+  link:https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/[FreeBSD
+  Handbook]. The User Handbook covers the fundamentals on how to use Jenkins as
+  well as content which is not explained in the *Tutorials* or *How-to Guides*
+  (above). This content is based predominantly on the "reference" description in
+  Daniele Procida's blog post
+  "link:https://www.divio.com/en/blog/documentation/[What nobody tells you about
+  documentation]" blog post, with appropriate "discussion"- (i.e.
+  background/overview material) and general "how-to guide"- (i.e. specific to
+  the chapter/topic in question) like material. Read more about
+  <<adding-user-handbook-content,Adding User Handbook content>>.
+* *Resources*:
+** The Pipeline Syntax Reference is a link to the published
+   link:content/doc/book/pipeline/syntax.adoc[syntax.adoc]
+   reference page in the *User Handbook*.
+** The Pipeline Steps Reference consists of Asciidoc files which are
+   auto-generated from content within the relevant Pipeline plugin source code.
+   Therefore, to contribute to this content, you need to edit the relevant
+   plugin's source code.
+* *Recent Tutorial Blog Posts* - these are a list of the most recently
+  published blog posts presented as tutorials (and tagged with the *tutorial*
+  tag).
+
+What about the *Guided Tour*?
```

Comment:
Thanks for pointing this out @bitwiseman - I deliberately worded this to be a rhetorical question because I wanted to draw people's attention to it (as a result of its clear omission from the list of parts in the *Jenkins User Documentation* section above).

The explanation about the fate of the *Guided Tour* follows this question - i.e. gradually decommissioning this part once there are sufficient Tutorials and How-to guides to cover the existing subject matter.

To make this more obvious (and improve the consistency/structure of this page), I'll make this question a bullet point and add the explanation following a dash - i.e. similar to the description of the other parts above.

Reply:
See [PR 1205](https://github.com/jenkins-infra/jenkins.io/pull/1205) for details.